docs/develop/persisting-add-in-state-and-settings.md

```diff
@@ -122,9 +121,9 @@ The anonymous function passed into the `saveAsync` method as the _callback_ para
 ## How to save custom XML to the document
 
 > [!NOTE]
-> This section discusses custom XML parts in the context of the Office Common JavaScript API which is supported in Word. The application-specific Excel JavaScript API also provides access to the custom XML parts. The Excel APIs and programming patterns are somewhat different. For more information, see [Excel CustomXmlPart](/javascript/api/excel/excel.customxmlpart).
+> This section discusses custom XML parts in the context of the Office Common JavaScript API which is supported in Word. The application-specific JavaScript APIs for Excel and for Word also provide access to the custom XML parts. The application-specific APIs and programming patterns are somewhat different from the Common version. For more information, see [Excel.CustomXmlPart](/javascript/api/excel/excel.customxmlpart) and [Word.CustomXmlPart](/javascript/api/word/word.customxmlpart).
 
-There is an additional storage option when you need to store information that exceeds the size limits of the document Settings or which has a structured character. You can persist custom XML markup in a task pane add-in for Word (and for Excel, but see the note at the top of this section). In Word, you use the [CustomXmlPart](/javascript/api/office/office.customxmlpart) object and its methods (again, see the note above for Excel). The following code creates a custom XML part and displays its ID and then its content in divs on the page. Note that there must be an `xmlns` attribute in the XML string.
+There's an additional storage option when you need to store information that exceeds the size limits of the document settings or which has a structured character. You can persist custom XML markup in a task pane add-in for Word (and for Excel and Word using application-specific API per the note at the beginning of this section). In Word, you can use the [CustomXmlPart](/javascript/api/office/office.customxmlpart) object and its methods (again, see the earlier note about application-specific APIs for Excel and Word). The following code creates a custom XML part and displays its ID and then its content in divs on the page. Note that there must be an `xmlns` attribute in the XML string.
```

Comment:
```suggestion
There's an additional storage option when you need to store information that exceeds the size limits of the document settings or which has a structured character. You can persist custom XML markup in a task pane add-in for Word (and for Excel and Word using application-specific API as mentioned in the previous paragraph). In Word, you can use the [CustomXmlPart](/javascript/api/office/office.customxmlpart) object and its methods. The following code creates a custom XML part and displays its ID and then its content in divs on the page. Note that there must be an `xmlns` attribute in the XML string.
```
I think two mentions of the previous note is unnecessary. If you are concerned that people might skip over the note, consider changing it to a paragraph and then say "as mentioned in the previous paragraph" instead.